sunpy/lightcurve/sources/stereo.py

```diff
@@ -0,0 +1,906 @@
+		data = _to_pandas(data)
+		
+		n = len(header)
+		return [OrderedDict(zip(range(n), header)), data]
+
+
+class SITLightCurve(LightCurve):
+	"""
+	SIT LightCurve. Provides SIT data back to 2007-07.
+	Most recent data is usually available one or two days late. 
+	Currently the LightCurve supports only Single File Load !!
+
+	Parameters
+	----------
+	timerange: sunpy.time.TimeRange
+		time range for which data is to be downloaded.
+		Example value -  TimeRange('2007-01-01','2015-03-01')   
+
+	stereo_spacecraft: string   
+		Default value - ahead
+		Possible values - ahead, behind    # corresponding to spacecraft location
+
+	specie:  string
+		Default value - 4He
+		Possible values - 4He, Fe, H, O
+
+	duration_of_average: string
+	Default value - 15min
+		Possible values - 1min, 10min, 1hr, 1day        #corresponding to duration over which data is averaged
+
+
+	Examples
+	--------
+	>>> import os
+	>>> import sunpy.data.test
+	>>> filepath = sunpy.data.test.rootdir
+	>>> from sunpy import lightcurve as lc
+	>>> sit = lc.SITLightCurve._parse_txt(os.path.join(filepath , 'sit/SIT_Ahead_10min_H_2007_01.txt'))
+	>>> sit = lc.SITLightCurve(sit[1],sit[0])
+	>>> sit.peek()
+
+
+	References
+	----------
+	| http://www.srl.caltech.edu/STEREO/Public/SIT_public.html
+
+	"""
+
+	def peek(self, title="SIT elemental Intensities"):
+		"""Plots SIT light curve in the usual manner"""
+
+		figure = plt.figure()
+		axes = plt.gca()
+
+		dates = matplotlib.dates.date2num(self.data['DateTime'].astype(datetime))
+
+		num_energy_bins = (len(self.header.values())-2)/2
+		colors = ['Green','Red','Chocolate', 'Blue','SeaGreen','Tomato','SlateBlue','Orange','Purple','Magenta','MediumVioletRed']
+
+		figure.delaxes(ax)
+		axes = figure.add_axes([0.1, 0.15, 0.55, 0.8])
+
+		for i,line in enumerate(self.header.values()):
+			if i >= 2 and i <= num_energy_bins + 1:
+				axes.plot_date(dates, self.data[line].ffill(), '-',
+					 label=line[1:20], color=colors[i-2], lw=0.5)
+		
+		axes.set_yscale("log",nonposy='mask')
+		axes.set_ylim(1e-3, 1e+3)
+		axes.set_title(title + ' : ' + self.header.values()[-1][:self.header.values()[-1].index(' ')])
+		axes.set_ylabel('1/(cm^2 s sr MeV/nuc)')
+		axes.set_xlabel('UTC TimeZone')
+
+
+		axes.yaxis.grid(True, 'major')
+		axes.xaxis.grid(False, 'major')
+		axes.legend(loc='center left', bbox_to_anchor=(1, 0.5))
+
+		figure.autofmt_xdate()
+		plt.show()
+
+		return figure
+
+
+	@staticmethod
+	def _parse_txt(filepath):
+		"""
+		Parses a STEREO SIT file from
+		http://www.srl.caltech.edu/STEREO/Public/SIT_public.html
+		
+		and returns header and Pandas.Dataframe containing data
+	
+		"""
+		header = []
+		# To read in column names
+		data_all = open(filepath)
+		for i, line in enumerate(data_all):
+			if i > 13 :
+				 header.append(line)
+			if line == 'BEGIN DATA\n':
+				data_start = i+1
+				break
+		data_all.close()
+
+		specie = header[-2][header[-2].index(':')+3:header[-2].index('t')-1]
```

Comment:
you could do:
```python
# To extract the species (4He) from a line like
# Columns 17 - 26:  4He total counts for same energy ranges as above
specie = header[-2].split(": ")[1].split()[0]
```
Also, it would be good to put a comment above with an example of a line, so people can understand what you are trying to do